immuni_common/helpers/sanic.py

```diff
@@ -178,3 +184,52 @@ def _validate_json_content_type(request: Request) -> None:
         return _wrapper
 
     return _decorator
+
+
+async def wait_configured_time() -> None:
+    """
+    Wait for the configured time.
+    This is usually useful to make dummy requests last a similar amount of time when compared to the
+    real ones, or slow down potential brute force attacks.
+    """
+    await asyncio.sleep(
+        random.normalvariate(
+            config.DUMMY_REQUEST_TIMEOUT_MILLIS, config.DUMMY_REQUEST_TIMEOUT_SIGMA
+        )
+        / 1000.0
+    )
+
+
+def handle_dummy_requests(
+    responses: List[WeightedPair],
```

Comment:
`WeightedPair` -> `WeightedResponse`?

Reply:
It doesn't really make much sense "WeightedResponse" (why would a response have a weight?). If you don't like "WeightedPair" maybe "WeightedTuple" would make more sense? 

I like this solution because it's generic, so if we ever need this mechanism another time we can use the same objects.

Reply:
The solution tries to be generic but actually isn't, IMHO. I see at least three reasons to support this:
(i) The keyword argument of this decorator is called `responses`, not `pairs`.
(ii) The "pair" is only ever used in responses context.
(iii) The named tuple itself has a `payload` field, which is not generic.

About the question "why would a response have a weight?": in this context, it actually does have a meaning, since a specific weight is being associated with a specific response (payload).
At the same time, `Pair` doesn't seem right either, since you are defining a pair of values, but only one—the payload(or response)—is weighted.

Reply:
> The keyword argument of this decorator is called responses, not pairs

That's because in the context of the decorator, we're trying to explain that the input must be a weightedPair of responses.

> The "pair" is only ever used in responses context.
🤔  yes, but that doesn't mean it has to be type specific

> The named tuple itself has a payload field, which is not generic
Why not? `Payload` seems generic enough to me, do you have a better suggestion for that name?

My intention was to make this as generic as possible (I also added the note with the mypy issue on why I couldn't make it truly generic). If you have a suggestion for a better name that makes it clear I'm all for it 😄